.github/workflows/publish.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish to BCR
+on:
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: git tag to publish
+        required: true
+        type: string
+jobs:
+  publish:
+    uses: bazel-contrib/publish-to-bcr/.github/workflows/publish.yaml@v0.2.3
+    with:
+      tag_name: ${{ inputs.tag_name }}
+      # GitHub repository which is a fork of the upstream where the Pull Request will be opened.
+      registry_fork: buildbuddy-io/bazel-central-registry
+      draft: false
```

Comment:
Should we start with draft==true first and then switch back?

I just want to avoid any auto-approve happening on BCR side